tools/kwiver_tool_runner.cxx

```diff
@@ -180,7 +180,7 @@ int main(int argc, char *argv[])
   kwiver::vital::plugin_manager_internal& vpm = kwiver::vital::plugin_manager_internal::instance();
 
   const std::string exec_path = kwiver::vital::get_executable_path();
-  vpm.add_search_path(exec_path + "/../lib/kwiver/plugins");
+  vpm.add_search_path(exec_path + "/../" KWIVER_LIBDIR "/kwiver/plugins");
```

Comment:
I know this change is mostly intended to support LIBDIR variants like `lib`, `lib64`, and `lib32`, but is it possible that `${CMAKE_INSTALL_BINDIR}` could be something other than a single directory (e.g. `bin/Release` on Windows) causing this up-one-level relative path to break?